ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,577 @@
+                      colorTheme: CardColorTheme,
+                      buttonTitle: String? = nil) {
+        self.primaryText = title
+        self.style = style
+        self.secondaryText = subtitle
+        self.icon = icon
+        self.image = image
+        self.buttonTitle = buttonTitle
+        self.colorTheme = colorTheme
+
+        super.init(frame: .zero)
+
+        configureSubviews()
+    }
+
+    @available(*, unavailable)
+    @objc public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    private func configureSubviews() {
+        translatesAutoresizingMaskIntoConstraints = false
+
+        // View border and background
+        layer.borderWidth = Constants.borderWidth
+        layer.cornerRadius = Constants.borderRadius
+        layer.borderColor = Constants.defaultBorderColor.cgColor
+        backgroundColor = Constants.defaultBackgroundColor
+
+        if style != .announcement {
+            let tapGesture = UITapGestureRecognizer(target: self, action: #selector(handleCardTapped(_:)))
+            addGestureRecognizer(tapGesture)
+        }
+
+        // Title
+        primaryLabel.text = self.primaryText
+        primaryLabel.numberOfLines = titleNumberOfLines
+        primaryLabel.textAlignment = style.textAlignment
+        addSubview(primaryLabel)
+
+        // Subtitle
+        if let secondaryText = secondaryText {
+            secondaryLabel.text = secondaryText
+            secondaryLabel.numberOfLines = subtitleNumberOfLines
+            secondaryLabel.textAlignment = style.textAlignment
+            addSubview(secondaryLabel)
+        }
+
+        // Icon
+        if let icon = icon {
+            let iconView = UIImageView(image: icon)
+            iconView.translatesAutoresizingMaskIntoConstraints = false
+            iconView.adjustsImageSizeForAccessibilityContentSizeCategory = true
+            self.iconView = iconView
+            addSubview(iconView)
+        }
+
+        // Featured Image
+        if let image = image {
+            let imageView = UIImageView(image: image)
+            imageView.translatesAutoresizingMaskIntoConstraints = false
+            self.imageView = imageView
+            addSubview(imageView)
+        }
+
+        // Card button - for style 'announcement' only
+        if style == .announcement {
+            if let buttonTitle = buttonTitle {
+                let button = Button(style: .borderless)
+                button.setTitle(buttonTitle, for: .normal)
+                button.addTarget(self, action: #selector(handleCardTapped(_:)), for: .touchUpInside)
+                button.translatesAutoresizingMaskIntoConstraints = false
+                self.button = button
+                addSubview(button)
+            }
+        }
+
+        setupLayoutConstraints()
+    }
+
+    open override func layoutSubviews() {
+        super.layoutSubviews()
+
+        setupColors()
```

Comment:
Does this really need to get called every time the subviews are laid out?
You should probably call this only in two places:
viewDidLoad and didMoveToWindow